e2e/test-plugins/grafana-extensionstest-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@test-plugins/extensions-test-app",
-  "version": "1.0.0",
```

Comment:
Is it a brand new package that will be published for the first time? 🤔

Reply:
I'm not sure but I got confirmation that this change is fine